[Bug] ccxt FileNotFoundError in initialize_network_objects

### Background

This bug is show-stopping for anyone doing any coding on pdr-backend. Therefore critical.

Have a fresh install of everything. If I run any unit test, or any `pdr` call, I get this error:

```text
venv/lib/python3.11/site-packages/ccxt/static_dependencies/ethereum/utils/network.py:34: in initialize_network_objects
    with open(
E   FileNotFoundError: [Errno 2] No such file or directory: '/Users/trentmc/code/pdr-backend/venv/lib/python3.11/site-packages/ccxt/static_dependencies/ethereum/utils/__json/eth_networks.json'
```

This is now happening in remote CI too. [Log](https://github.com/oceanprotocol/pdr-backend/actions/runs/8173250309/job/22348368651)

Easiest way to reproduce: from console, go to python and import ccxt.


### Full traceback

```text
...
pdr_backend/cli/arg_feed.py:4: in <module>
    import ccxt
venv/lib/python3.11/site-packages/ccxt/__init__.py:29: in <module>
    from ccxt.base.exchange import Exchange                     # noqa: F401
venv/lib/python3.11/site-packages/ccxt/base/__init__.py:24: in <module>
    from ccxt.base import exchange
venv/lib/python3.11/site-packages/ccxt/base/exchange.py:57: in <module>
    from ccxt.static_dependencies.ethereum import abi
venv/lib/python3.11/site-packages/ccxt/static_dependencies/ethereum/__init__.py:1: in <module>
    from .abi import *
venv/lib/python3.11/site-packages/ccxt/static_dependencies/ethereum/abi/__init__.py:2: in <module>
    from .abi import (
venv/lib/python3.11/site-packages/ccxt/static_dependencies/ethereum/abi/abi.py:1: in <module>
    from .codec import (
venv/lib/python3.11/site-packages/ccxt/static_dependencies/ethereum/abi/codec.py:12: in <module>
    from ..utils import (
venv/lib/python3.11/site-packages/ccxt/static_dependencies/ethereum/utils/__init__.py:100: in <module>
    from .network import (
venv/lib/python3.11/site-packages/ccxt/static_dependencies/ethereum/utils/network.py:61: in <module>
    networks = initialize_network_objects()
venv/lib/python3.11/site-packages/ccxt/static_dependencies/ethereum/utils/network.py:34: in initialize_network_objects
    with open(
E   FileNotFoundError: [Errno 2] No such file or directory: '/Users/trentmc/code/pdr-backend/venv/lib/python3.11/site-packages/ccxt/static_dependencies/ethereum/utils/__json/eth_networks.json'
```

--- a/setup.py
+++ b/setup.py
@@ -10,7 +10,7 @@ from setuptools import find_packages, setup
 # or pip install -e .
 install_requirements = [
     "bumpversion",
-    "ccxt>=4.1.59",
+    "ccxt==4.1.59",
     "coverage",
     "enforce_typing",
     "eth-account==0.11.0",

